src/components/ManageDataModal.js

```diff
@@ -76,6 +94,33 @@ export default function ManageDataModal({ show, onClose }) {
                             )}
                         </div>
                     </Tab>
+                    <Tab eventKey="runelite" title="Runelite Import">
+                        <div className="text-center mt-3">
+                            <Alert variant="info" className="small">
+                                The OsLeague plugin from the Runelite Plugin Hub is required.<br/>
```

Comment:
Minor thing, can you replace this `<br/>` with `<p></p>` tags? 

```suggestion
                                <p>The OsLeague plugin from the Runelite Plugin Hub is required.</p>
```

Had a weird crash issue the other day that was caused by using `<br/>` in react, apparently it's an outdated way of adding space.

Reply:
Added. I originally didn't like the extra margin it was adding with `<p>` tags. I wound up adding `mb-0` to that `<p>`.

Reply:
Yeah, that works, or you can just wrap it in a `<div>` instead of `<p>`. Either is fine